### DISCUSSION TITLE
I cant show the name of the school and my names and date of expiration date in one image

### DISCUSSION THREAD

**Discussion by PiponcheDev** [community/community#142652] [GitHub Education]
### Hi everyone,

In my country our school IDs are like little books on the cover its the name and address of the school and then when you open it is all the other information like a date of expiration and my full three emails and also all this is in bulgarian. How can i post two picks and how to make it so the bot previewing my images reads what its on there.

**Comment by Aninooo** [comment]
Take Clear Photos - Make sure the images of your school ID are well-lit and in focus. This helps improve readability for any OCR (Optical Character Recognition) software.

Use an OCR Tool -  Before posting, you can use an OCR tool to extract the text from your images. There are many free online tools and apps that can convert images to text.

Post Images - Depending on the platform you're using, you can typically attach images by clicking on an "attach" or "upload" button. Some platforms may allow you to drag and drop the images directly.

Add Text Description - When you post the images, include a text description or context about what’s in the images. This can help both users and bots understand the content.
